Assets/Scripts/UI/InGame/Graphics/GraphicsController.cs

```diff
@@ -1,173 +1,125 @@
-﻿using UnityEngine;
+﻿using System;
+using UnityEngine;
 using UnityEngine.UI;
-using System;
 
 namespace Assets.Scripts.UI.InGame
 {
-	public class GraphicsController : MonoBehaviour {
+    public class GraphicsController : MonoBehaviour {
 
-		public GeneralGraphics GeneralGraphic;
-		public QualitySwitcher QualitySwitcher;
+        public Button SaveGraphicPrefs;
+        public Button LoadGraphicPrefs;
+        public Button DeleteGraphicPrefs;
+        public Text PrefsLabel;
+
+        public GeneralGraphics GeneralGraphic;
 		public ResolutionSwitcher ResolutionSwitcher;
-		public FPSLimiter FPSLimiter;
-		public Text label;
-		public Toggle CustomSettings;
 
-		// Index of the custom quality level
-		public const int CUSTOM = 6;
-		public const string GDATA = "GraphicsData";
+        public const string GDATA = "GraphicsData";
 		public const string QDATA = "QualityProfile";
-		public const string FDATA = "FPSLimit";
+        public const string FDATA = "FPSLimit";
 
-		private void Start()
+        protected void Start()
```

Comment:
Why was this changed to protected?

Reply:
because visual studio was having an attitude about it being private since it hides the inherited method, and I am obviously a slave to VS whims and wishes :)